src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx

```diff
@@ -161,22 +150,18 @@ const TargetCategorySelect: React.FC<{
   );
 };
 
-const TargetNameSelect: React.FC<{ cluster: Cluster | null }> = ({
-  cluster,
-}) => {
+const TargetNameSelect: React.FC = () => {
   const { t } = useTranslation();
 
   const dispatch = useDispatch();
   const reportData = useSelector((state: IRootState) => state.report.data);
   const targetName = reportData.target.name;
-
-  let targetNameOptions: string[]; // decided by floor and targetCategory
+  const targetNameWithCategory = targetNameSelections;
+  let targetNameOptions: string[]; // decided by targetCategory
 
   if (reportData.floor && reportData.target.category) {
-    targetNameOptions = getOptions(cluster?.data.name || "").targetName.find(
```

Comment:
Seems like `getOptions`  is not  used now, remember to remove it.

Reply:
Ok, I remove the createOptions.ts file